modules/core_api/fsw/inc/cfe_es_api_typedefs.h

```diff
@@ -83,6 +83,18 @@ typedef CFE_ES_TaskEntryFuncPtr_t CFE_ES_ChildTaskMainFuncPtr_t;
  */
 typedef void *CFE_ES_StackPointer_t; /* aka osal_stackptr_t in proposed OSAL change */
 
+/**
+ * \brief Checksum/CRC algorithm identifiers
+ *
+ * Currently only CFE_ES_CRC_16 is supported.
+ */
+typedef enum CFE_ES_CrcType
+{
+    CFE_ES_CRC_8,  /**< \brief CRC ( 8 bit additive - returns 32 bit total) (Currently not implemented) */
+    CFE_ES_CRC_16, /**< \brief CRC (16 bit additive - returns 32 bit total) */
+    CFE_ES_CRC_32  /**< \brief CRC (32 bit additive - returns 32 bit total) (Currently not implemented) */
+} CFE_ES_CrcType_t;
+
```

Comment:
To adhere to the enum naming convention, the type should be named `CFE_ES_CrcType_Enum_t` and the labels should be `CFE_ES_CrcType_CRC{_ 8,_16,_32}`  (i.e. consistent use of CFE_ES_CrcType_ prefix for both type and labels, with same capitalization)

Reply:
No worries - I've updated them in the 2nd commit.

Reply:
Looks good, but one more observation is that the original enum values were 1, 2, and 3 (so CRC_16 was 2).  We should keep the values the same, just in case.

Reply:
Yes no worries - I was thinking about that. Will update them now to start at 1.

Reply:
Almost there, just a nuance in the GSFC coding standards that says we need to be explicit about every enum label in this case.  In other words, even though ISO C guarantees that an enum label without a value will be assigned the value of the previous label plus 1, the local coding standards say that we must also put this in the source code.

So just add explicit values for 2 and 3 here....